check-smtp/lib/check-smtp.go

```diff
@@ -0,0 +1,120 @@
+package checksmtp
+
+import (
+	"os"
+
+	"net/smtp"
+
+	"fmt"
+
+	"crypto/tls"
+
+	"net"
+
+	"time"
+
+	"github.com/jessevdk/go-flags"
+	"github.com/mackerelio/checkers"
+)
+
+var opts struct {
+	Host     string  `short:"H" long:"host" default:"localhost" description:"Hostname"`
+	Port     string  `short:"p" long:"port" default:"25" description:"Port"`
+	FQDN     string  `short:"F" long:"fqdn" description:"FQDN used for HELO"`
+	SMTPS    bool    `short:"s" long:"smtps" description:"Use SMTP over TLS"`
+	StartTLS bool    `short:"S" long:"starttls" description:"Use STARTTLS"`
+	Auth     string  `short:"A" long:"authmech" description:"SMTP AUTH Authentication Mechanisms (only PLAIN supported)"`
+	User     string  `short:"U" long:"authuser" description:"SMTP AUTH username"`
+	Password string  `short:"P" long:"authpassword" description:"SMTP AUTH password"`
+	Warning  float64 `short:"w" long:"warning" default:"3" description:"Warning threshold (sec)"`
+	Critical float64 `short:"c" long:"critical" default:"5" description:"Critical threshold (sec)"`
```

Comment:
I think it's not intuitive that the plugin has default thresholds. User should explicitly specify the monitoring thresholds. What do you think?

Reply:
I think it is correct your concern.
For that reason deleted default value of options of threshold.